cmd/fetchd/cmd/gendelegations.go

```diff
@@ -36,6 +56,35 @@ func AddGenesisDelegationCmd(defaultNodeHome string) *cobra.Command {
 
 			config.SetRoot(clientCtx.HomeDir)
 
+			minDelegatedAmountStr, err := cmd.Flags().GetString(flagMinDelegatedAmount)
+			if err != nil {
+				return fmt.Errorf("failed to get flag %q: %w", flagMinDelegatedAmount, err)
+			}
+			accountReservedAmountStr, err := cmd.Flags().GetString(flagAccountReservedAmount)
+			if err != nil {
+				return fmt.Errorf("failed to get flag %q: %w", flagAccountReservedAmount, err)
+			}
+
+			minDelegatedCoin, err := sdk.ParseCoinNormalized(minDelegatedAmountStr)
```

Comment:
@ejfitzgerald this `ParseCoinNormalized` will truncate amount decimals:  
`873355710070564895.14afet` becomes `873355710070564895afet`

This seems the regular way to parse coin amount input, but not sure about losing some precision here, wdyt?

Reply:
Yep totally fine. Technically speaking there is no such thing as fractial afet so this truncation is fine